module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -2732,49 +2712,149 @@ function ($matches) {
         return $msg;
     }
 
+    /**
+     * Get a cached record and check that it has the requested fields
+     *
+     * @param string $cacheId Cache entry ID
+     * @param array  $fields  Fields
```

Comment:
Maybe adding a word to this description would be helpful -- e.g. Requested fields or Required fields?

Reply:
I revised this and other descriptions as well.